src/Nemo.jl

```diff
@@ -651,4 +651,10 @@ include("Deprecations.jl")
 
 include("Native.jl")
 
+function Base.isapprox(x::T, y::T;
```

Comment:
All is green but this is a bit of a hack, not sure we should include it, perhaps better to instead wait for https://github.com/Nemocas/AbstractAlgebra.jl/pull/1843 ?

Reply:
yeah, better to wait